docs/docs/cmd/viva/engage/engage-community-add.mdx

```diff
@@ -0,0 +1,170 @@
+import Global from '/docs/cmd/_global.mdx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# viva engage community add
+
+Creates a new community in Viva Engage.
+
+## Usage
+
+```sh
+m365 viva engage community add [options]
+```
+
+## Options
+
+```md definition-list
+`--displayName <displayName>`
+: The name of the community. The maximum length is 255 characters.
+
+`--description <description>`
+: The description of the community. The maximum length is 1024 characters.
+
+`--privacy <privacy>`
+: Defines the privacy level of the community. The possible values are: public, and private.
+
+`--adminEntraIds [adminEntraIds]`
+: Comma-separated list of Microsoft Entra IDs, assigning admin privileges to the designated individuals. Required when using application permissions. Specify either adminEntraIds or adminEntraUserNames, but not both.
+
+`--adminEntraUserNames [adminEntraUserNames]`
+: Comma-separated list of Microsoft Entra UPNs, assigning admin privileges to the designated individuals. Required when using application permissions. Specify either adminEntraIds or adminEntraUserNames, but not both.
+
+`--wait`
+: Wait for the operation to finish.
+```
+
+<Global />
+
+## Remarks
+
+:::warning
+
+This command is based on an API that is currently in preview and is subject to change once the API reached general availability.
+
+:::
+
+It is limited to networks in Native mode only - legacy and external Yammer networks will not be able to use this API for community creation.
+
+Creating a community will create a linked Microsoft 365 unified group as well.
+
+In order to use this command, you need to grant the Microsoft Entra application used by the CLI for Microsoft 365 the permission to the Viva Engage API. To do this, execute the `cli consent --service VivaEngage` command.
+
+## Examples
+
+Create a public community and wait for the community to be created
+
+```sh
+m365 viva engage community add --displayName "Software engineers" --description "A community for all software engineers" --privacy public --wait
+```
+
+Create a private community
+
+```sh
+m365 viva engage community add --displayName "Software engineers" --description "A community for all software engineers" --privacy private
+```
+
+Create a private community with owners
+
+```sh
+m365 viva engage community add --displayName "Software engineers" --description "A community for all software engineers" --privacy private --adminEntraUserNames "john.doe@contoso.onmicrosoft.com,jane.doe@contoso.onmicrosoft.com"
+```
+
+## Response
+
+### With --wait:
```

Comment:
When listing multiple responses, we always have a default response, and other responses. Check `planner plan set` as example.